docs/copilot/copilot-extensibility-overview.md

```diff
@@ -10,34 +10,46 @@ MetaSocialImage: images/shared/github-copilot-social.png
 
-- [Tutorial: Create a code tutor chat participant with the Chat API](/api/extension-guides/chat-tutorial.md)
-- [Tutorial: Generate code annotations by using the Language Model API](/api/extension-guides/language-model-tutorial.md)
+## Use cases
 
-## Related content
+You can extend the capabilities of Copilot in VS Code in various ways. For example, you could use them for:
+
+- Docs querying: A chat participant could allow Copilot Chat to query a third-party documentation service to find information about a specific topic.
+- AI-assisted coding: Use the Copilot LLM to provide editor annotations to provide coding suggestions.
+- AI-powered reviews: Use the Copilot LLM to review your code for security vulnerabilities or performance improvements.
+- Data retrieval: A chat participant could allow Copilot Chat to query a database or third-party data service to retrieve information about a specific topic.
+
+## Get started with Copilot extensibility in VS Code
+
+To get started with extending Copilot in your VS Code extension, explore the following resources:
 
-- [VS Code Chat API reference](/api/extension-guides/chat.md)
-- [VS Code Language Model API reference](/api/extension-guides/language-model.md)
+- **Tutorial: AI-powered code annotations**: step-by-step guide to implement a VS Code extension that uses the Language Model API to [generate code annotations](/api/extension-guides/language-model-tutorial.md) in the editor to help improve your code.
 
-- [Copilot Chat extension sample](https://github.com/microsoft/vscode-extension-samples/tree/main/chat-sample)
-- Make sure to watch our Build session about [Enhancing VS Code extensions with GitHub Copilot](https://www.youtube.com/watch?v=YI7kjWzIiTM) to learn all about these APIs.
+- **Tutorial: Code tutor chat participant**: step-by-step guide to implement a [code tutor chat participant](/api/extension-guides/chat-tutorial.md) that enables users to ask for explaining a technical topic by using natural language in the Chat view in VS Code.
 
-- [GitHub Copilot extensions are all you need](https://code.visualstudio.com/blogs/2024/06/24/extensions-are-all-you-need) blog post
+- **Chat participant sample**: sample code for getting started with building a [VS Code Chat extension](https://github.com/microsoft/vscode-extension-samples/tree/main/chat-sample) that uses the Chat API and Language Model API.
+
+- **Extension guides**: Learn how to use the [Chat API](/api/extension-guides/chat.md) and [Language Model API](/api/extension-guides/language-model.md) to extend Copilot in your VS Code extension.
+
+## Related content
 
-- Join the [Copilot Partner Program](https://github.com/features/preview/copilot-partner-program) to extend GitHub Copilot with a GitHub App
-- Watch our [Build session](https://www.youtube.com/watch?v=RXaLlCeaBIA) about extending GitHub Copilot to see it in action.
+- [Get started with Copilot Extensions](https://github.com/features/copilot/extensions)
```

Comment:
Should we link to the chat docs, and the language model docs as related content?

Reply:
Good point. Thanks